docs/error-reporting/platform-integrations/ios/setup.md

```diff
@@ -44,9 +44,23 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 - Xcode 10 or above
 
 ## Install the SDK
+You can use this SDK through either Swift Package Manager or CocoaPods. The SPM package can be integrated directly within Xcode or by editing your package's Package.swift file.<br>
+Choose one of the following integration methods.
 
-Use [CocoaPods](https://cocoapods.org/) to install the latest version of the reporting library. To use CocoaPods, add the [Backtrace pod](https://cocoapods.org/pods/Backtrace) to your `Podfile`:
+### Via Xcode
```

Comment:
@rick-bt ~ Is Xcode the default installation method?

Reply:
No, there is not a default installation method.

Reply:
Thanks!